Solitaire.java

```diff
@@ -167,12 +167,23 @@ public static boolean playAgain(Scanner s){
 	* @param e the formatted array from format()
 	*/
 
-	public static void encrypt(String[] e) {
-		//convert letters to numbers
-		//get keystream values
-		//add the numbers together
-		//if the added numbers are greater than 26, subtract by 26
-		//convert back to letters
+	public static void encrypt(String[] e, int [] key) {
+		int[] intArray = new int[e.length()];
```

Comment:
in this case, e.length() should be e.length, because length() is for use on strings while length is for arrays

Reply:
same goes for the couple of other .lengths() in lines 172 thru 180